scripts/raid-size-tests.py

```diff
@@ -89,36 +92,41 @@ class FakeDev:
 
 
 def verify_size_ok(level, sizes):
-    devs = create_devices_for_sizes(sizes)
-    raid = create_raid(level, devs)
-    devs = [FakeDev(size) for size in sizes]
-    calc_size = get_raid_size(level, devs)
-    real_size = get_real_raid_size(raid)
-    if len(set(sizes)) == 1:
-        sz = '[{}]*{}'.format(humanize_size(sizes[0]), len(sizes))
-    else:
-        sz = str([humanize_size(s) for s in sizes])
-    print("level {} sizes {} -> calc_size {} real_size {}".format(
-        level, sz , calc_size, real_size), end=' ')
-    if calc_size > real_size:
-        print("BAAAAAAAAAAAD", real_size - calc_size)
-        r = False
-    else:
-        print("OK by", real_size - calc_size)
-        r = True
-    run(['mdadm', '--verbose', '--stop', raid])
-    raids.remove(raid)
+    r = False
+    try:
+        devs = create_devices_for_sizes(sizes)
+        raid = create_raid(level, devs)
+        devs = [FakeDev(size) for size in sizes]
+        calc_size = get_raid_size(level, devs)
+        real_size = get_real_raid_size(raid)
+        if len(set(sizes)) == 1:
+            sz = '[{}]*{}'.format(humanize_size(sizes[0]), len(sizes))
+        else:
+            sz = str([humanize_size(s) for s in sizes])
+        print("level {} sizes {} -> calc_size {} real_size {}".format(
+            level, sz , calc_size, real_size), end=' ')
+        if calc_size > real_size:
+            print("BAAAAAAAAAAAD", real_size - calc_size)
+        else:
+            print("OK by", real_size - calc_size)
```

Comment:
print("RAID overhead %s bytes" % real_size - calc_size)

Reply:
That's not the RAID overhead though, it's the difference between our estimate of the size and the real size. I'll make this a bit clearer.